@codex please fix

```
Run npm run lint

> dspace@3.0.0 lint
> cd frontend && npm run lint


> dspace@3.0.0 lint
> npm run lint:a11y && cross-env ESLINT_USE_FLAT_CONFIG=false npx eslint . --max-warnings=0


> dspace@3.0.0 lint:a11y
> node scripts/a11y-lint.mjs

(node:2517) ESLintRCWarning: You are using an eslintrc configuration file, which is deprecated and support will be removed in v10.0.0. Please migrate to an eslint.config.js file. See https://eslint.org/docs/latest/use/configure/migration-guide for details. An eslintrc configuration file is used because you have the ESLINT_USE_FLAT_CONFIG environment variable set to false. If you want to use an eslint.config.js file, remove the environment variable. If you want to find the location of the eslintrc configuration file, use the --debug flag.
(Use `node --trace-warnings ...` to show where the warning was created)

> dspace@3.0.0 type-check
> tsc --noEmit

Error: scripts/tests/ensurePlaywrightBrowsers.test.ts(65,25): error TS2339: Property 'NODE_OPTIONS' does not exist on type '{ HTTPS_PROXY: string; npm_config_https_proxy: string; TZ?: string; }'.
Error: Process completed with exit code 2
```

new codex task, not a r/e/v/i/e/w task

--- a/scripts/tests/ensurePlaywrightBrowsers.test.ts
+++ b/scripts/tests/ensurePlaywrightBrowsers.test.ts
@@ -57,7 +57,7 @@ describe('ensurePlaywrightBrowsers', () => {
     let chromeExists = false;
     let depsSentinelExists = false;
     const existingHeadless = new Set<string>();
-    const envWithProxy = {
+    const envWithProxy: NodeJS.ProcessEnv = {
       ...process.env,
       HTTPS_PROXY: 'http://legit-proxy:3128',
       npm_config_https_proxy: 'http://legit-proxy:3128',

